src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandler.java

```diff
@@ -57,4 +74,34 @@ protected void updateCaseStatus(SscsCaseData caseData) {
             caseData.setInterlocReviewState(InterlocReviewState.AWAITING_ADMIN_ACTION);
         }
     }
+
+    private void handlePostHearingReview(SscsCaseData caseData,
+                                        CcdCallbackMap action,
+                                        PostHearingReviewType reviewType,
+                                        String userAuthorisation) {
+
+        DocumentLink previewDocument = caseData.getDocumentStaging().getPreviewDocument();
+        DocumentType documentType = getDocumentTypeFromReviewType(reviewType);
+        String actionName = action.getCcdDefinition();
+        String docFileName = action.getCallbackSummary() + ".pdf";
+
+        SscsDocumentDetails documentDetails = SscsDocumentDetails.builder()
+            .documentDateAdded(LocalDate.now().toString())
+            .documentType(documentType.getValue())
+            .documentFileName(docFileName)
+            .documentLink(previewDocument)
+            .build();
+
+        SscsDocument document = new SscsDocument(documentDetails);
+        List<SscsDocument> documents = new ArrayList<>();
+        documents.add(document);
+
+        SscsUtil.addDocumentToBundle(footerService, caseData, documents.get(0), docFileName);
+
+        EventType eventType = getEventTypeFromDocumentReviewTypeAndAction(reviewType, actionName);
+        String templateId = documentConfiguration.getDocuments()
+            .get(caseData.getLanguagePreference())
+            .get(eventType);
+        issueDocument(caseData, documentType, templateId, generateFile, userAuthorisation);
```

Comment:
does this actually issue the document? To me it does seem like it makes the document but then adds it as the preview document which field we clear on line 63

Reply:
Done. Doc won't be getting cleared anymore.